src/transformers/modeling_outputs.py

```diff
@@ -1622,3 +1623,40 @@ class SampleTSPredictionOutput(ModelOutput):
     """
 
     sequences: torch.FloatTensor = None
+
+
+@dataclass
+class MaskedImageModelingOutput(ModelOutput):
+    """
+    Base class for outputs of masked image completion / in-painting models.
+
+    Args:
+        loss (`torch.FloatTensor` of shape `(1,)`, *optional*, returned when `labels` is provided):
```

Comment:
(nit) models like ViTForMaskedImageModeling don't require `labels` to return a loss, but rather `bool_masked_pos`. So perhaps we can leave out "returned when labels is provided"